src/main/scala/xiangshan/frontend/PreDecode.scala

```diff
@@ -275,9 +275,9 @@ class RVCExpander(implicit p: Parameters) extends XSModule {
   })
 
   if (HasCExtension) {
-    io.out := new RVCDecoder(io.in, XLEN, useAddiForMv = true).decode
+    io.out := new RVCDecoder(io.in, XLEN, XLEN, useAddiForMv = true).decode
```

Comment:
A minor question: is the current design defaulting to `FLEN=XLEN`? Why does `trait HasXSParameter` define `val fLen = 64` but it seems never used?

Apart from this, the IFU/Predecode part LGTM.

Reply:
Thank you for your reminder. Using fLen is better here.